src/node_jsvmapi.h

```diff
@@ -18,6 +18,7 @@
 #ifndef SRC_NODE_JSVMAPI_H_
 #define SRC_NODE_JSVMAPI_H_
 
+#include "node.h"
```

Comment:
As a follow on we may want to refactor node.h so that is not the case because as Jason says we want to avoid leakage of the v8 apis into modules building against the NAPI api.